### PR TITLE
Add model testing capabilities

### PR DESCRIPTION
**Pull request by sbotor**

### CODE REVIEW

File: lenet.py [hunk 1]
```diff
@@ -14,10 +15,11 @@ def __init__(self):
         self.fc1 = nn.Linear(in_features=120, out_features=84)
         self.fc2 = nn.Linear(in_features=84, out_features=_NUM_CLASSES)
```
Comment (MrChrisFix) [inline lenet.py]: Not sure if this should't be the RBF function.

File: lenet.py [hunk 2]
```diff
@@ -14,10 +15,11 @@ def __init__(self):
...
 
-        self.pool = nn.AvgPool2d(kernel_size=2)
+        self.pool = nn.MaxPool2d(kernel_size=2, stride=2)
         self.activ = nn.Tanh()
```
Comment (MrChrisFix) [inline lenet.py]: I think the Tanh() function must be multiplied by the 'A' constant equal to 1.7159

File: testing.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                images = images.to(DEVICE)
+                labels = labels.to(DEVICE)
+
+                outputs = self.model(images)
+                _, pred = torch.max(outputs.data, 1)
+                total += labels.size(0)
+                correct += (pred == labels).sum().item()
+
+        acc = 100 * correct / total
+        print(f'Accuracy: {format_accuracy(acc)}% ({correct}/{total})')
```
Comment (MrChrisFix) [inline testing.py]: Maybe we could add other statistics such as recoll and precision?

**Review by MrChrisFix — CHANGES_REQUESTED** [review]

File: testing.py (hunk 3 above)
Comment (MrChrisFix) [reply testing.py]: Resolved: Will be added in future

File: lenet.py (hunk 1 above)
Comment (sbotor) [reply lenet.py]: Maybe later

**Review by MrChrisFix — APPROVED** [review]
LGTM

**Pull request merged**